swift/apple/FirezoneKit/Sources/FirezoneKit/Views/SettingsView.swift

```diff
@@ -644,15 +642,16 @@ public struct SettingsView: View {
   }
 
   func saveSettings() async throws {
+    try await viewModel.applySettingsToStore()
+
     do {
```

Comment:
Should we get rid of this `do` block now? How does the caller handle exceptions?

Reply:
Good catch, I think we have a global error handler in the UI that catches anything that makes it there. This error is pretty rare - if we hit it, no IPC calls are likely to work.

It would be possible to hit this if:

- they open the settings window
- they remove the VPN configuration from system settings
- they try to save or export logs etc

Probably not worth solving at this point.

Reply:
> I think we have a global error handler in the UI that catches anything that makes it there.

That is something worth verifying as it would likely simplify a lot of error handling the UI code.